docs/taskruns.md

```diff
@@ -254,18 +254,23 @@ For more information, see the [`LimitRange` code example](../examples/v1beta1/ta
 
 ## Configuring the failure timeout
 
-You can use the `timeout` field to set the `TaskRun's` desired timeout value in minutes.
-If you do not specify this value in the `TaskRun`, the global default timeout value applies.
-If you set the timeout to 0, the `TaskRun` fails immediately upon encountering an error.
+You can use the `timeout` field to set the `TaskRun's` desired timeout value. If you do not specify this 
+value for the `TaskRun`, the global default timeout value applies. If you set the timeout to 0, the `TaskRun` will 
+have no timeout and will fail immediately upon encountering an error.
```

Comment:
i'm nitpicking wording that isn't yours but "the `TaskRun` will have no timeout and will fail immediately upon encountering an error" confused me a bit b/c "fail immediately upon encountering an error" is true regardless of whether there is a timeout or not

i think what this might want to say is something like "the `TaskRun` will  have no timeout and will run until it completes either successfully or with an error"?

Reply:
Thanks! I changed to the following:

```
If you set the global timeout to 0, all `TaskRuns` that do not have a timeout set 
will have no timeout and will run until it completes successfully or fails from an error.
```